pkpdapp/pkpdapp/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,2 @@
+Someone asked for password reset for email {{ email }}. Follow the link below:
```

Comment:
should this not extend the base.html?

Reply:
maybe for consistency, but I'm not a fan of lots of html in emails, so thought it would be better to be as plain as possible...?

Reply:
Sure let's just leave it like that.